### PR TITLE
DW-579: Updated email handler default bodies

### PR DESCRIPTION
**Pull request by jekuaitk**
https://jira.itkdev.dk/browse/DW-579

* Adds `OS2Forms Selvbetjening` module to allow setting a description for the message body section of the email handler configuration.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,6 +127,18 @@ $config['os2forms_get_organized'] = [
...
 ```
 
+### Selvbetjening Module
+
+The `OS2Forms Selvbetjening` module updates the Webform Email Handler
+by adding a description to the message body section. The
+description should be configured in the `settings.local.php` file:
+
+```php
+$config['os2forms_selvbetjening']['email_body_description'] = "Brug enten standardsvaret eller definer dit eget svar. Se <a href=\"https://os2forms.os2.eu/mail-tekster\">OS2Forms Loop</a> for andre standarder og eksempler.";
```
Comment (rimi-itk) [inline README.md:137]: Use `'` to simplify the description content (`\` can be removed).

File: README.md [hunk 2]
```diff
@@ -127,6 +127,18 @@ $config['os2forms_get_organized'] = [
...
+```
+
+If it is not, no description is added.
```
Comment (rimi-itk) [inline README.md:140]: Are you missing a “set” here, i.e. should it read “If it is not set, …”?

File: web/modules/custom/os2forms_selvbetjening/os2forms_selvbetjening.info.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+name: 'OS2Forms Selvbetjening'
+type: module
+description: 'Provides changes to webform email handler.'
+package: 'OS2Forms'
+core: 8.x
+core_version_requirement: ^8 || ^9
```
Comment (rimi-itk) [inline web/modules/custom/os2forms_selvbetjening/os2forms_selvbetjening.info.yml:6]: Remove `^8 ||` – Drupal 8 is not supported anymore.

**Review by rimi-itk — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jekuaitk) [reply README.md:137]: Updated.

File: README.md (hunk 2 above)
Comment (jekuaitk) [reply README.md:140]: Updated.

File: web/modules/custom/os2forms_selvbetjening/os2forms_selvbetjening.info.yml (hunk 3 above)
Comment (jekuaitk) [reply web/modules/custom/os2forms_selvbetjening/os2forms_selvbetjening.info.yml]: Removed.

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**